typescript/src/api-doc-types/OverloadInfo.ts

```diff
@@ -12,8 +12,10 @@ import { ParameterDoc } from './ParameterDoc';
  */
 export class OverloadInfo extends ExportDoc implements ParameterContainer {
```

Comment:
I wonder if `OverloadInfo` should extend `BaseApiDoc` rather than `ExportDoc`?

Reply:
Yeah. I agree with that. Since it's part of the `DocCollection` it might be also checked using `instanceof ExportDoc` and therefore shouldn't extend from that.